volume/store/store.go

```diff
@@ -302,16 +302,14 @@ func (s *VolumeStore) Dereference(v volume.Volume, ref string) {
 
 	s.globalLock.Lock()
 	defer s.globalLock.Unlock()
-	refs, exists := s.refs[v.Name()]
-	if !exists {
-		return
-	}
```

Comment:
I think this check is still need. Or else if `s.refs[v.Name()]` does not exists, the `s.refs[v.Name()] = refs` will make it exists, this may cause issue if some other operation relay on `refs, exists := s.refs[v.Name()]`